dbt/adapters/dremio/relation.py

```diff
@@ -37,8 +37,10 @@ class DremioIncludePolicy(Policy):
 
 @dataclass(frozen=True, eq=False, repr=False)
 class DremioRelation(BaseRelation):
-    quote_policy: DremioQuotePolicy = DremioQuotePolicy()
-    include_policy: DremioIncludePolicy = DremioIncludePolicy()
+    quote_policy: DremioQuotePolicy = field(default_factory=lambda: DremioQuotePolicy())
```

Comment:
Just out of curiosity, why this change?

Reply:
It's a requirement for supporting python 3.11. See https://github.com/dbt-labs/dbt-core/pull/6326 for more info.

Reply:
For those interested: https://stackoverflow.com/questions/52063759/passing-default-list-argument-to-dataclasses